Open_Acidification_pH-stat_arduino/startsWith.ino

```diff
@@ -4,4 +4,4 @@ bool startsWith(const char *a, const char *b) {
   if (strncmp(a, b, strlen(b)) == 0)
```

Comment:
While this function was pulled from an online resource. We can rename it to match our naming standard. We would not want to rename it if the code was in a downloaded library file.